app/lib/shared/utils.dart

```diff
@@ -238,6 +238,8 @@ Future<http.Response> getUrlWithRetry(http.Client client, String url,
         return result;
       }
     } catch (e, st) {
+      print(e);
+      print(st);
```

Comment:
If this is for testing only, let's move to `test/utils.dart`

Reply:
leftover debug statement, removing